dev/standalone/integration-test-dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python
+
+WORKDIR /app/
+
+# Install test requirements first to make running the tests faster
+SHELL ["/bin/bash", "-c"] 
+COPY integration_requirements.txt /app/integration_requirements.txt
+RUN pip install virtualenv && \
+    virtualenv /tmp/gng_testing && \
+    source /tmp/gng_testing/bin/activate &&\
+    pip install -r integration_requirements.txt
```

Comment:
Thinking a bit more about it, you don't need a virtualenv inside a docker container, the container is already an isolated env.

```suggestion
RUN pip install -r integration_requirements.txt
```

Reply:
the test script creates a virtual env, so that's why this is here.